architecture/infrastructure_components/kubernetes_infrastructure.adoc

```diff
@@ -86,20 +71,20 @@ configuration files].
 
 === Kubelet
 
-The kubelet on each node updates the node as specified by a container manifest,
-which is a YAML file that describes a pod. The kubelet uses a set of manifests
-to ensure that the described containers are started and that they continue to
-run. A sample manifest can be found in the
+Each node has a kubelet that updates the node as specified by a container
+manifest, which is a YAML file that describes a pod. The kubelet uses a set of
+manifests to ensure that its containers are started and that they
+continue to run. A sample manifest can be found in the
 https://cloud.google.com/compute/docs/containers/container_vms#container_manifest[Kubernetes
 documentation].
 
 There are a number of ways that a container manifest can be provided to a
 kubelet:
 
-- A file path on the command line that is checked every 20 seconds.
-- An HTTP endpoint passed on the command line that is checked every 20 seconds.
-- The kubelet watching an *etcd* server, such as *_/registry/hosts/$(hostname -f)_*, and acting on any changes.
-- The kubelet listening for HTTP and responding to a simple API to submit a new
+- A file path on the command line is checked every 20 seconds.
+- An HTTP endpoint passed on the command line is checked every 20 seconds.
+- The kubelet watches an *etcd* server, such as *_/registry/hosts/$(hostname -f)_*, and acts on any changes.
+- The kubelet listens for HTTP and responds to a simple API to submit a new
  manifest.
```

Comment:
These changes make it sound less like these are "ways that it _can_ be provided". Not sure if it's a big deal though.